stock_delivery_request_report/models/stock_picking.py

```diff
@@ -14,6 +14,24 @@ class StockPicking(models.Model):
     delivery_due_report_date = fields.Date(
         compute='_compute_report_delivery_due_date',
     )
+    scheduled_date = fields.Date(
+        string='Scheduled DateTime'
+    )
+    scheduled_date_custom = fields.Date(
+        string='Scheduled Date',
+        compute='_compute_scheduled_date',
```

Comment:
Let's change the method name to `_compute_scheduled_date_custom` as the current name conflicts with the standard method here https://github.com/odoo/odoo/blob/b8c7eddb3493e881a3e2c3d41bd0166ff93e4ef9/addons/stock/models/stock_picking.py#L386.